less/about/preferences.less

```diff
@@ -1,10 +1,14 @@
 @import "./common.less";
 
-strong, div, span, li, em, p, a {
+div, span, li, em, p, a {
   font-weight: 400;
   -webkit-font-smoothing: antialiased;
 }
 
+strong {
+  font-weight: 600
```

Comment:
wouldn't this change cause a visual regression?

Reply:
I don't think so, only other `<strong>` tag we have is in URL bar, which is not affected by this

Reply:
I see. the file will eventually be removed so not a great issue anyway.